ci/tiledb-java-final-jar.yml

```diff
@@ -71,4 +71,12 @@ steps:
       GPG_SECRET_KEYS_ENC: $(GPG_SECRET_KEYS_ENC)
       SONATYPE_USERNAME: $(SONATYPE_USERNAME)
       SONATYPE_PASSWORD: $(SONATYPE_PASSWORD)
-    displayName: "Publish"
\ No newline at end of file
+    displayName: "Publish"
+
+  - task: GithubRelease@0
```

Comment:
Let's do this in a separate PR, that way the build.grade change is clean.

Reply:
done